Improve Process Readme (#282)
w3c/wot-usecases: USE-CASES/README.md

# Basic Flow of Use Case Submission

## How to submit use cases

> 1. Please submit your use cases to [this directory](https://github.com/w3c/wot-usecases/tree/main/USE-CASES) using [Pull Request](https://github.com/w3c/wot-usecases/pulls) or by using a GitHub issue using the [Markdown template](use-case-template.md) or the [HTML template](use-case-template.html) (The HTML template is preferred.).

Reviewer: ```suggestion
1. Please submit your use cases to using a GitHub issue with the provided Use Case template.
```

>   - A Pull Request is preferred if you are a member of the WoT IG or WG. For external collaborators, issues are the only way to submit Use Cases.

Reviewer: ```suggestion
```

> 2. Make sure to use <https://w3c.github.io/wot-usecases/#definitions> when formulating your use case.

Reviewer: ```suggestion
```

## After approval by the Use Cases TF

> 1. Use cases that have been reviewed and approved for inclusion in [the IG note](https://w3c.github.io/wot-usecases/) are moved to https://github.com/w3c/wot-usecases/tree/master/USE-CASES/processed for archival purposes.

Reviewer: ```suggestion
Use cases that have been reviewed and approved for inclusion in [the IG note](https://w3c.github.io/wot-usecases/) are closed but not deleted for archival purposes.
```
